### ISSUE TITLE
Fix nav order and Play Area/Control Area split

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#72]
Since https://github.com/phetsims/ratio-and-proportion/issues/58, this has gotten off, and it would be good to work out the right order.

**Comment by zepumph** [comment]
Steps for this issue:

1. Solidify the UI components for each screen (I think this is done, but would be good to double check with a designer)
2. Get a list of what elements are in the play area vs control area (likely this will be done in the design doc.
3. implement the set order according to the design.

Over to @emily-phet until this is ready for implementation.

**Comment by emily-phet** [comment]
Nav order should be as listed in the nav order table in the description design doc.

Everything sim specific is in the play area *except* the reset all button.

**Comment by zepumph** [comment]
Implemented above.

Dev note: I ran into trouble with this because I was mutating like `pdomPlayAreNode.accessibleOrder.push()` instead of setting accessibleOrder each time. This ended up not calling `PDOMTree.accessibleOrderChange` because it only mutated the value. This broke the DAG PDOM rules we had in place, and confused me for some time. To help fix this, I wrote a note about the accessibleOrder getter and also wrote a lint rule to prevent pushing values on `accessibleOrder`. I want to mention this to @jessegreenberg tomorrow because it was a gotcha, and to see if he like the way I tried to steer future devs (most likely future me) in the right direction.

**Comment by zepumph** [comment]
I noted in the a11y dev meeting, closing
